changelog/unreleased/issue-6389.toml

```diff
@@ -0,0 +1,5 @@
+type = "a"
+message = "Added ability to add notifications to illuminate event definitions"
+
+issues = ["6389"]
+pulls = []
```

Comment:
Since this change only affects enterprise Illuminate users, I think we can remove the changelog from this PR and just add this PR to the preexisting changelog [here](https://github.com/Graylog2/graylog-plugin-enterprise/blob/master/changelog/unreleased/issue-6388.toml)